### PR TITLE
Feature/signupimprovements

### PR DESCRIPTION
**Pull request by mikebashford**
Ready to go:

- Left side set to spec of figma mock up
- SVG imported as a component to change the colors
- Gradient button added for sign up

### CODE REVIEW

File: public/images/SignupGraphic.svg
Comment (coltonehrman) [inline public/images/SignupGraphic.svg:1]: We can delete this file now

File: src/app/SignUp/page.tsx
Comment (coltonehrman) [inline src/app/SignUp/page.tsx:1]: Can we fix this folder name to be lowercase?

**Review by coltonehrman — COMMENTED** [review]
Left some comments

File: public/images/SignupGraphic.svg
Comment (mikebashford) [reply public/images/SignupGraphic.svg:1]: Deleted

File: src/app/SignUp/page.tsx
Comment (mikebashford) [reply src/app/SignUp/page.tsx:1]: I've set it in my local copy to be lower case.

Comment (coltonehrman) [reply src/app/SignUp/page.tsx:1]: Are you able to commit and push?

Comment (mikebashford) [reply src/app/SignUp/page.tsx:1]: Should be good now, definitely some weirdness with renaming a folder with case sensitivity.

**Review by coltonehrman — APPROVED** [review]
LGTM!

**Pull request merged**